fineract-provider/src/main/java/org/apache/fineract/commands/domain/CommandWrapper.java

```diff
@@ -38,6 +38,8 @@
     private final String json;
```

Comment:
This file should not be part of pull request as it is having only formatting issue